.travis.yml

```diff
@@ -1,40 +1,74 @@
 language: cpp
 sudo: false
 
-os:
-- linux
-- osx
-
-compiler:
-- clang
-- gcc
+# Use Linux by default
+os: linux
 
 matrix:
-  exclude:
-  - os: osx
-    compiler: gcc
-
-addons:
-  apt:
-    sources:
-    - ubuntu-toolchain-r-test
-    packages:
-    - g++-4.8
-    - gdb
-
-
-before_script:
-- if [ "${TRAVIS_OS_NAME}" = "osx" ]; then
-    brew update;
-    brew install gdb;
-  fi
+  include:
+    # AppleClang
+    # OSX workers are slower to start up.  Having these first in the build matrix makes travis faster overall.
+    - os: osx
+      compiler: clang
+      env: &Release
+        - TRAVIS_CUBERITE_BUILD_TYPE=RELEASE CUBERITE_PATH=./Cuberite
+    - os: osx
+      compiler: clang
+      env: &Debug
+        - TRAVIS_CUBERITE_BUILD_TYPE=DEBUG CUBERITE_PATH=./Cuberite_debug
+    # Default clang
+    - compiler: clang
+      addons: &gdb
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gdb
+      env: *Release
+    - os: linux
```

Comment:
Why is os explicitly specified here, when it is not specified elsewhere?

Reply:
Just an oversight from when I added the default.  Fixed now.